src/gridfs/download.ts

```diff
@@ -44,10 +44,16 @@ export interface GridFSFile {
   length: number;
   chunkSize: number;
   filename: string;
-  contentType?: string;
-  aliases?: string[];
   metadata?: Document;
   uploadDate: Date;
+  /**
+   * @deprecated Will be removed in the next major version.
+   */
+  contentType?: string;
```

Comment:
minor nit: Since we don't have any other inline documentation here, we can have the deprecation notice be on one line.
```suggestion
  /** @deprecated Will be removed in the next major version. */
  contentType?: string;
```

Reply:
Updated.